rfcs/openapi_generator/generate.py

```diff
@@ -36,6 +36,11 @@
 from llama_toolchain.stack import LlamaStack
 
 
+STREAMING_ENDPOINTS = [
+    "/agentic_system/turn/create"
```

Comment:
Let us make sure to annotate the endpoints instead of listing endpoints in the generator. We wouldnt want to change the generator when we add another endpoint.

Reply:
@raghotham yes, the right fix needs fixing our openapi generator. I will at least make sure our openapi generator code (which is forked) is completely present in the repository now.

Reply:
But for the moment let's get this in so @dltn is unblocked.